src/listeners/command-handler/CoreMessageParser.ts

```diff
@@ -72,6 +72,36 @@ export class CoreListener extends Listener<typeof Events.PreMessageParsed> {
 		return null;
 	}
 
+	private getManagedRoleMentionPrefix(message: Message): string | null {
+		// If the message does not originate in a guild then skip early
+		if (!message.guild) return null;
+
+		// If there is no user on the client then skip early
+		if (!this.container.client.user) return null;
```

Comment:
But like..you can't get a message without the client user o.o

Reply:
Code has been changed by Kyra and me. Please re-review